docs/pages/preview/eas-update/how-eas-update-works.md

```diff
@@ -0,0 +1,75 @@
+- Runtime version: The runtime version describes the JS–native interface defined by the native code layer that runs our app's update layer. It is defined in **app.json** or **app.config.js**. Whenever we make changes to our native code that change our app's JS–native interface, we'll need to update the runtime version.
+- Platform: Every build has a platform, such as "Android" or "iOS".
+
+If we were to make two builds with the channel named "staging" and two builds with the channel named "production", we'd end up with something like this:
+
+![Build types diagram](/static/images/eas-update/builds.png)
+
+This diagram is just an example of how you could create builds and name their channels, and where you could put those builds. Ultimately it's up to you which channel names you set and where you put those builds.
+
+**Publishing an update**
+
+After we've created builds, we can change the update layer of our project and publish it as an update, which will make our changes available to certain builds. For example, we could change some text inside **App.js**, and then we could publish that change as an update.
+
+To publish an update, we can run `eas branch:publish --auto`. This command will create a local update bundle inside the **dist/** folder in our project. Once it's created an update bundle, it will upload that bundle to EAS' servers, in a database object named a _branch_. A branch has a name, and contains a list of updates, where the most recent update is the active update on the branch.
+
+![Branches with its most recent update pointed out as the active one](/static/images/eas-update/branch.png)
+
+**Matching updates and builds**
+
+Like builds, every update on a branch includes a target runtime version and target platform. With these fields, we can make sure that an update will run on a build with something called an _update policy_. EAS' update policy is as follows:
+
+- The platform of the build and the target platform of an update must match exactly.
+- The runtime version of the build and the target runtime version of an update must match exactly.
+- A channel can be linked to any branch. By default, a channel is linked to a branch of the same name.
+
+Let's focus on that last point. Every build has a channel, and we, as developers, can link that channel to any branch, which will make its most recent compatible update available on the channel. To simplify this linking, by default we'll auto-link a channel to a branch of the same name. For instance, if we created builds with the channel named "production", we could publish updates to a branch named "production" and our builds would get the updates on a matching branch named "production", even though we did not manually link anything.
```

Comment:
In some of the other pages, branch names are code-formatted. Whatever we decide, let's format them consistently unless there's an intentional reason to do otherwise:
```suggestion
Let's focus on that last point. Every build has a channel, and we, as developers, can link that channel to any branch, which will make its most recent compatible update available on the channel. To simplify this linking, by default we'll auto-link a channel to a branch of the same name. For instance, if we created builds with the channel named "production", we could publish updates to a branch named `production` and our builds would get the updates on a matching branch named `production`, even though we did not manually link anything.
```